circle.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
      working_directory: /var/www
      docker:
-        - image: retreatguru/php56-apache
+        - image: retreatguru/php71-apache
```

Comment:
this is tricky, PHP is so darn backwards compatible. I'm not sure how to best test the plugin. 
use the old docker image to make sure it works for old installs (zilliions) or use the latest php 7.4 and ensure it works for new installs. but then run the risk we put in some new php code like ?? that will fail in 5.6. 
I think we have a test to ensure our code still works for old versions. ...

maybe just keep it simple and don't change anything?

Reply:
yeah, that sounds like a good reason to me to keep it simple.
You're right, people will still be running websites with php 5.6, 15 years from now

Reply:
but I did notice in composer.json that we have a test setup for it, just not sure when it runs. but I recall writing this because i got bitten once!

it's fine to deploy this code as is.